ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml

```diff
@@ -55,7 +55,10 @@ tests:
     env:
       BASE_DOMAIN: perfscale.rox.systems
       COMPUTE_NODE_REPLICAS: "24"
+      E2E_REPOSITORY: https://github.com/davdhacs/e2e-benchmarking
```

Comment:
Just cut a new kube-burner-ocp version v1.4.0 including the patch for metrics-profiles, you can override the kube-burner version for this step with the env var KUBE_BURNER_VERSION https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner-ocp-wrapper/run.sh#L11C23-L11C42